gildniy/WayFarer
# #167254059 Ft create a trip

#### What does this PR do?
Create a trip and make it available to the users
#### Description of Task to be completed?
As a user of role admin, a user will be able to create a trip by selecting:
- Departure place
- Destination place
- Trip date
- Trip time
- Bus Plate Number
#### How should this be manually tested?
By cloning the site or navigating to the GitHub page: [https://gildniy.github.io/WayFarer/UI](url) a landing page Two buttons appear at the landing page 'TEST AS USER' and 'TEST AS ADMIN', these will allow entering the site. And if the user enters as an Admin, the Dashboard page will have a list of all bookings, and below there is a button 'CREATE A TRIP' by clicking on it, a form to create a trip will appear. And can be only submitted by filling all the field.
The Destination and the Departure place cannot be identical in selection.
#### What are the relevant pivotal tracker stories?
#167254059
